pkg/operator/resources/syncapi/dashboard.go

```diff
@@ -79,18 +79,24 @@ func addAPIToDashboardObject(dashboard *aws.CloudWatchDashboard, dashboardName s
 	dashboard.Widgets = append(dashboard.Widgets, aws.TextWidget(1, highestY+1, 22, 1, "## "+apiName))
 
 	var grid aws.CloudWatchWidgetGrid
-	grid.Init(1, highestY+2, 6, 11, 2)
+	err = grid.FillNewGridVertically(1, highestY+2, 6, 11, 3)
```

Comment:
How do you think about this instead of lines 81+82:

```go
err, grid := NewVerticalGrid(1, highestY+2, 6, 11, 3)
```

Reply:
Yep, totally makes sense.